src/pg_to_evalscript/javascript_processes/array_concat.js

```diff
@@ -0,0 +1,14 @@
+function array_concat(arguments) {
+  const { array1, array2 } = arguments;
+  if (array1 === null || array1 === undefined) {
+    throw new Error(
+      "Mandatory argument `array1` is either null or not defined."
+    );
+  }
+  if (array2 === null || array2 === undefined) {
```

Comment:
Perhaps it would also make sense to check if both arguments are arrays. fftr

Reply:
Yeah that makes sense. Added and will also add for #15

Reply:
:+1: